modules/microshift-multus-intro.adoc

```diff
@@ -6,21 +6,60 @@
 [id="microshift-multus-intro_{context}"]
 = Additional networks in {microshift-short}
 
-During cluster installation, the _default_ pod network is configured with default values unless you customize the configuration. The default network handles all ordinary network traffic for the cluster. Using the Multus CNI plugin, you can add additional interfaces to pods from other networks. This gives you flexibility when you configure pods that deliver network functionality, such as switching or routing.
+During cluster installation, the _default_ pod network is configured with default values unless you customize the configuration. The default network handles all ordinary network traffic for the cluster. Using the {microshift-short} Multus CNI plugin, you can add additional interfaces to pods from other networks. This gives you flexibility when you configure pods that deliver network functionality, such as switching or routing.
 
-[id="additional-network-use-cases"]
+[id="microshift-supported-additional-networks_{context}"]
+== Supported additional networks for network isolation
+The following additional networks are supported in {microshift-short} {product-version}:
+
+* Bridge: Allows pods on the same host to communicate with each other and the host.
+
+* IPVLAN: Allows pods on a host to communicate with other hosts.
+** This similar to a MACVLAN-based additional network.
```

Comment:
Should macvlan be listed first if we're referring to it here?

Reply:
The list is alphabetical. Be easier to expand in the future that way as well.